src/Microsoft.AspNetCore.Proxy/ProxyExtensions.cs

```diff
@@ -1,114 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Proxy;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Options;
-
-namespace Microsoft.AspNetCore.Builder
-{
-    public static class ProxyExtensions
-    {
-        /// <summary>
-        /// Runs proxy forwarding requests to the server specified by base uri.
-        /// </summary>
-        /// <param name="app"></param>
-        /// <param name="baseUri">Destination base uri</param>
-        public static void RunProxy(this IApplicationBuilder app, Uri baseUri)
```

Comment:
We can't remove these methods as it would be a breaking change.

Reply:
Proxy has never shipped, breaking changes aren't a problem.